app/config/Config.scala

```diff
@@ -52,6 +54,11 @@ class Config(playConfig: Configuration) extends AwsInstanceTags with Logging {
     case _ => Set(atomWorkshopUrl)
   }
 
+  private lazy val recipesUrls: Set[String] = stage match {
+    case Code => Set(s"https://recipes.${Dev.appDomain}", recipesUrl) // allow recipes in DEV to call Workflow CODE
+    case _ => Set(recipesUrl)
+  }
+
```

Comment:
Now that we are doing this three times, it might be clearer to refactor into a function?

If not, while you are here can you correct the comment for `atomWorkshopUrls` which says that it allows **MAM** in DEV to call workflow CODE?

Reply:
good spot, PR refactored now